server/graphql/v2/scalar/DateString.ts

```diff
@@ -0,0 +1,10 @@
+import { GraphQLScalarType } from 'graphql';
+
+const DateString = new GraphQLScalarType({
+  name: 'DateString',
+  serialize: value => {
+    return value.toString();
+  },
+});
+
+export { DateString };
```

Comment:
Same remark for .ts extension

Reply:
I didn't see any comments related to `.ts` files.

Reply:
I'm not sure where it went!

It was saying I would prefer to not use .ts extension if we're not using typescript within the file.

Reply:
But won't this cases problems with the ts engine. For example I suppose that the ts engine is not analyzing and compiling js files. So, we are probably losing type safety when we import js files into ts files and vice versa. I am not expert in ts that is just a humble guess.

Reply:
Out of consistency, let's keep it as .js for now and we can bring this topic in a separated issue. So we can quietly look at the situation and take a clear informed decision.

Reply:
Done. I found [this article](https://mariusschulz.com/blog/type-checking-javascript-files-with-checkjs-in-typescript) about the subject.